tests/Feature/Controllers/MutateUpdateOperationsTest.php

```
@@ -2258,6 +2404,61 @@ public function test_updating_a_resource_with_attaching_multiple_belongs_to_many
         );
     }
 
+    public function test_updating_a_resource_with_attaching_empty_belongs_to_many_relation(): void
+    {
+        $modelToUpdate = ModelFactory::new()->createOne();
+        $belongsToManyRelationToAttach1 = BelongsToManyRelationFactory::new()
```

PHPMD (2.15.0)

[warning] Avoid unused local variables such as '$belongsToManyRelationToAttach1'. (undefined)

(UnusedLocalVariable)